scripts/blinky

```diff
@@ -12,6 +12,7 @@ primary.add_argument("-S", action='store_true', default=False, dest='install', h
 primary.add_argument("-Sr", action='store_true', default=False, dest='rebuildpkg', help="Reinstall package(s) from AUR (including rebuild)")
 primary.add_argument("-Srr", action='store_true', default=False, dest='rebuildtree', help="Reinstall package(s) and dependencies from AUR (including rebuild)")
 primary.add_argument("-Ss", action='store_true', default=False, dest='search', help="Search for package(s) in AUR")
+primary.add_argument("-complete", action='store_true', default=False, dest='complete', help="completion helper")
```

Comment:
we should set help to `argparse.SUPPRESS` to make the completion-flag vanish from `--help`.

https://docs.python.org/3/library/argparse.html#help

Reply:
TIL!